### DISCUSSION TITLE
Newb Install Method...

### DISCUSSION THREAD

**Discussion by tomWhilbert** [Security-Onion-Solutions/securityonion#1709] [General] [answered]
Never used the tool before and want to install it in a VM to explore.  What method would you'all recommend?

1.  Full Centos ISO install?
2. Install on top of an Ubuntu 18.04 server (already have one of those easy to deploy in my home esxi).
3. Use Docker?  Am familiar with docker but not clear on the how I use it with Onion?
    Install on top of linux and then pull in needed components via docker?

Thanks for any input.

**Comment by dougburks** [comment, marked as answer]
Hi @tomWhilbert ,

For first time users, we highly recommend that you use our Security Onion ISO image as it's the quickest and easiest way to get started.

For more information, please see https://docs.securityonion.net/en/2.3/installation.html.

Thanks!

**Reply by tomWhilbert** [reply]
Thanks for the quick reply.  Thats what I did initially.  Just wanted to check to see if i was on the best track.

Thanks!
